samples/client/petstore/python/petstore_api/model_utils.py

```diff
@@ -1682,7 +1682,7 @@ def model_to_dict(model_instance, serialize=True):
                else:
                    res = []
                    for v in value:
-                       if isinstance(v, PRIMITIVE_TYPES) or v is None:
+                       if isinstance(v, (dict,) + PRIMITIVE_TYPES) or v is None:
```

Comment:
What if the dict contains model instances?
How about converting its values too?